wormhole-connect/vite.config.ts

```diff
@@ -91,7 +91,6 @@ const plugins = [
 const optimizeDeps = {
   include: [
     '@emotion/react',
-    '@emotion/styled',
```

Comment:
This is interesting, because I added all others in this list but `@emotion/styled` back in this [commit](https://github.com/wormhole-foundation/wormhole-connect/commit/8723059a33659ac4e59b75b7cb723a695e6b9cdc#diff-3a553c64c1df40f5df57b9d42834d24ba48a240cb80f257f66d3e14af1db3fb8). Maybe the complete fix was to add others and remove `@emotion/styled` 🤔 💡